Dockerfile

```diff
@@ -39,14 +44,19 @@ FROM dependencies as build
 COPY --chown=node:node . .
 RUN npm run build
 
-# ---- Release ----
-FROM base as release
+# ---- App ----
+FROM base as app
 
-COPY --from=build /usr/bin/dumb-init /usr/bin/dumb-init
 COPY --chown=node:node --from=build /home/node/app/dist .
 COPY --chown=node:node --from=build /home/node/app/prisma prisma
 COPY --chown=node:node --from=build /home/node/app/prod_node_modules node_modules
 
+# ---- Rrelase ----
+FROM base as release
```

Comment:
don't we need libssl install here too?

Reply:
It's installed in the base layer, and all other layers derive from it.